tests/autoyast/installation.pm

```diff
@@ -96,6 +101,15 @@ sub run {
                 @needles = grep { $_ ne 'autoyast-confirm' } @needles;
                 $confirmed = 1;
             }
+            elsif (match_has_tag('autoyast-license') || match_has_tag('inst-betawarning')) {
```

Comment:
I don't like this - I would rather it look for the beta warning if BETA=1

We do not want to allow the product to ship showing the beta warning when BETA!=1

(I almost let that happen once... Ralf caught it..very embarrasing..)